master/internal/resourcemanagers/agent_resource_manager.go

```diff
@@ -2,11 +2,17 @@ package resourcemanagers
 
 import (
 	"crypto/tls"
+	"strings"
 
 	"github.com/pkg/errors"
 
+	"github.com/determined-ai/determined/proto/pkg/apiv1"
```

Comment:
nonblocking: this line should be merged with the last couple of lines of importing